docs/developer/file-structure.rst

```diff
@@ -148,18 +118,16 @@ The PyMedPhys Source Code
 -------------------------
 
 Almost all users will access the PyMedPhys library of source code via the
-main `pymedphys` package (``pymedphys/src/pymedphys/``). However no library
-code actually exists within ``pymedphys/src/pymedphys/``. Instead, library code
-is contained within ``pymedphys/packages/`` and redirected through
-``pymedphys/src/pymedphys/`` via a set of python imports.
-
-Within ``pymedphys/packages/``, code is organised into a set of subpackages,
-such as ``pymedphys_analysis`` and ``pymedphys_dicom``. From there, each
-subpackage contains a directory named ``src/<package_name>/``. Within each
-``src/<package_name>/``, code is further arranged into categories, such as
-``gamma`` and ``mudensity``. These correspond to Python modules. Finally, code
-within these category directories is organised into levels. Levels define the
-dependency hierarchy of code within modules. See diagram below:
+main ``pymedphys`` package, located in
+``pymedphys/packages/pymedphys/src/pymedphys/``. However, no *library* code
+actually exists within this directory. Instead, all library code is arranged
+into PyMedPhys' various subpackages, such as ``pymedphys_analysis`` and
+``pymedphys_dicom``. From there, each subpackage contains a directory named
+``src/<package_name>/``. Within each ``src/<package_name>/``, code is further
+arranged into categories, such as ``gamma`` and ``mudensity``. These correspond
+to Python modules. Finally, code within these categorised directories is
+organised into *levels*. Levels define the dependency hierarchy of code within
```

Comment:
Levels are no longer being defined or exposed in any way within the file structure. Instead, the levelling is achieved apart from the file structure in the graphviz graphs....